### DISCUSSION TITLE
Nerdminer v2 on a USB Dongle with a ESP32-Pico-D4 4MB chip?

### DISCUSSION THREAD

**Discussion by abcx4711** [BitMaker-hub/NerdMiner_v2#516] [Q&A]
Hi, 
does it run on a USB Dongle / Development Board with a ESP32-Pico-D4 4MB chip?
https://www.aliexpress.com/item/1005007188412615.html
Regards

**Comment by Jaycetti81** [comment]
Did you ever find an answer?  I have 10 of them I would love to use as a Nerdminer

**Comment by abcx4711** [comment]
No

Am 24. März 2025 17:55:22 GMT-05:00 schrieb Jaycetti81 ***@***.***>:
